pyproject.toml

```diff
@@ -14,6 +14,7 @@ extend-exclude = [
 line-length = 120
 target-version = "py38"
 # See Rules in Ruff documentation: https://docs.astral.sh/ruff/rules/
+[tool.ruff.lint]
```

Comment:
Would you mind opening a separate MR about this?

Reply:
Do you want to revert these changes?

Reply:
No, I was seconds late with my review. No reason to revert if it's merged already.